discovery-provider/src/model_validator.py

```diff
@@ -0,0 +1,72 @@
+import json
+# import logging  # pylint: disable=C0302
+from jsonschema import validate, Draft7Validator, ValidationError, validators
+
+# logger = logging.getLogger(__name__)
+
+# https://app.quicktype.io/ -- JSON schema generator
+class ModelValidator:
+    # <model: {fields: [], schema: object}>
+    schema_and_fields_dict = {}
+
+    @staticmethod
+    def init_schema(model):
+         # If validation schema has been loaded before, refer to dict
+        if model in ModelValidator.schema_and_fields_dict:
+            schema = ModelValidator.schema_and_fields_dict[model]['schema']
+        else:
+            model_filename = './src/schema_' + model + '_simple_oneof.json'
+            with open(model_filename) as f:
+                schema = json.load(f)
+
+            ModelValidator.schema_and_fields_dict[model] = {"fields": schema['fields'], "schema": schema}
```

Comment:
What's the reason behind making this a dict vs making it the schema `ModelValidator.schema_and_fields_dict[model] = schema` and removing the `fields` key.

Reply:
i use the `fields` key in the validation in `models.py` [here](https://github.com/AudiusProject/audius-protocol/pull/487/files/425dcdd6ca037152c5180ad2467383c60ad64b87#diff-293c92d36be22235787bda7cf963eed0R166). i suppose this method name isn't the most clear on also adding the `fields` key